tests/test_backend.py

```diff
@@ -236,42 +262,303 @@ def test_get_best_backend_for_collections_basic(self, multi_backend_connection,
+            ],
+            "keywords": ["xcube", "SAR"],
+            "version": "0.8.0",
+            "deprecated": False,
+            "license": "proprietary",
+            'links': [
+                {"rel": "license", "href": "https://propietary.license"},
+            ],
+            "providers": [{"name": "provider2"}],
+            "datasource_type": "2sentinel-2-grd",
+            "sci:citation": "Second citation list."
+        })
 
         catalog = AggregatorCollectionCatalog(backends=multi_backend_connection)
         metadata = catalog.get_collection_metadata("S2")
         assert metadata == {
-            "stac_version": "0.9.0",
             "id": "S2",
             "description": "S2",
             "title": "b1's S2",
+            "stac_version": "1.0.0",
+            "stac_extensions": [
+                "datacube",
+                "sar",
+                "https://stac-extensions.github.io/datacube/v1.0.0/schema.json",
+                "https://stac-extensions.github.io/scientific/v1.0.0/schema.json",
+                "https://stac-extensions.github.io/sat/v1.0.0/schema.json",
+                "https://stac-extensions.github.io/sar/v1.0.0/schema.json",
+                "https://stac-extensions.github.io/raster/v1.0.0/schema.json"
+            ],
+            "crs": ["http://www.opengis.net/def/crs/OGC/1.3/CRS84", "http://www.opengis.net/def/crs/EPSG/0/2154"],
+            "keywords": ["S2", "Sentinel Hub", "xcube", "SAR"],
+            "version": "1.1.0",
+            "deprecated": False,
             "extent": {"spatial": {"bbox": [[-180, -90, 180, 90]]}, "temporal": {"interval": [[None, None]]}},
-            "license": "proprietary",
+            "license": "various",
             "summaries": {"provider:backend": ["b1", "b2"]},
-            "links": [],
+            "links": [
+                {"rel": "license", "href": "https://special.license.org"},
+                {"rel": "license", "href": "https://propietary.license"},
+                {'href': 'http://oeoa.test/openeo/1.1.0/collections', 'rel': 'root'},
+                {'href': 'http://oeoa.test/openeo/1.1.0/collections', 'rel': 'parent'},
+                {'href': 'http://oeoa.test/openeo/1.1.0/collections/S2', 'rel': 'self'}
+            ],
+            "providers": [{"name": "provider1"}, {"name": "provider2"}],
+            "type": "Collection",
+            "sci:citation": "Modified Copernicus Sentinel data [Year]/Sentinel Hub",
+        }
+
+    def test_get_collection_metadata_merging_summaries(
+        self, multi_backend_connection, backend1, backend2, requests_mock, flask_app
+    ):
+        requests_mock.get(backend1 + "/collections", json={"collections": [{"id": "S2"}]})
+        requests_mock.get(backend1 + "/collections/S2", json={
+            "id": "S2", "summaries": {
+                "constellation": ["sentinel-1"], "instruments": ["c-sar"], "platform": ["sentinel-1a", "sentinel-1b"],
+                "raster:bands": [{
```

Comment:
where does this "raster:bands" example come from, it does seem to follow anything from https://github.com/stac-extensions/raster#raster-band-object